src/config.js

```diff
@@ -242,7 +242,7 @@ global.config = {
   },
 
   market: {
-    minAmountToSell: 100000,
+    minAmountToSell: 50000,
```

Comment:
Same here, I think we should explain why we set a specific value / change to a specific value in the config.

Reply:
same here, my terminal we full with energy at 80k+ and i had no space

Reply:
no only energy at 😄

Reply:
Mind to write this as a comment to the line, so that we know in the future why we went in this direction?